### PR TITLE
[UK] Create uk translation of en/lessons/basics/comprehensions.md

### PR DESCRIPTION
**Pull request by NadiaGubernator**

### CODE REVIEW

File: uk/lessons/basics/comprehensions.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+version: 1.1.0
+title: Вирази
+---
+
+Спискові вирази - це синтаксичний цукор для проходження по перечисленням в Elixir.
```
Comment (doroshenkoss) [inline uk/lessons/basics/comprehensions.md]: This translation is awesome! Thanks! But maybe we can find more accurate traslation for `list comprehension`? For example - `Генераторні списки` [(Wiki: Генераторні списки)](https://uk.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BA%D0%BE%D0%B2%D1%96_%D0%B2%D0%B8%D1%80%D0%B0%D0%B7%D0%B8)

Comment (doroshenkoss) [reply uk/lessons/basics/comprehensions.md]: Requesting @brain-geek advice.

Comment (NadiaGubernator) [reply uk/lessons/basics/comprehensions.md]: Thanks for your kind words :) 
About list comprehensions - I see that Ukrainian authors have different opinions on this matter (i.e. https://uk.wikibooks.org/wiki/%D0%9F%D0%BE%D1%80%D0%B8%D0%BD%D1%8C%D1%82%D0%B5_%D1%83_Python_3/%D0%92%D0%B8%D1%80%D0%B0%D0%B7%D0%B8_%D0%BD%D0%B0%D0%B4_%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B0%D0%BC%D0%B8#%D0%A1%D0%BF%D0%B8%D1%81%D0%BA%D0%BE%D0%B2%D1%96_%D0%B2%D0%B8%D1%80%D0%B0%D0%B7%D0%B8). I should read more of the book I've mentioned earlier to understand if I like the style of technical translation there, but I'm inclined to agree that `comprehension` and `генератор` don't seem to have much in common.

Comment (doroshenkoss) [reply uk/lessons/basics/comprehensions.md]: Yep, you right. Let's keep `cпискові вирази`. I'll merge it.

**Pull request merged**